src/shared/selectMenu/SelectMenu.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import styles from './SelectMenu.module.scss';
+import classNames from 'classnames';
+import { ReactComponent as Chevron } from 'assets/icons/Chevron-Top.svg';
+
+export default ({ items }) => {
+  const [selected, setSelected] = useState(items[0].label);
+  const [showMenu, setShowMenu] = useState(false);
+
+  function getMenu() {
+    return (
+      <div
+        className={classNames(styles.Menu, !showMenu ? styles.MenuHide : '')}
+      >
+        {items.map((i, index) => (
+          <button
+            key={index}
+            className={styles.Option}
+            onClick={() => {
+              i.onClick();
```

Comment:
Given that items is some special array an propType specification would be pleasant.

Reply:
Yes, true. Added it.